src/test/scala/com/dwolla/cloudflare/DnsRecordClientSpec.scala

```diff
@@ -65,9 +65,45 @@ class DnsRecordClientSpec(implicit ee: ExecutionEnv) extends Specification with
       )).await
     }
 
+    "accept a domain name and content and return existing record" in new Setup {
+      mockGetZoneId("dwolla.com")
+      mockGetDnsRecords("fake-zone-id", "name=example.dwolla.com?content=example.dwollalabs.com", SampleResponses.Successes.listDnsRecordsWithOneResult)
```

Comment:
> `name=example.dwolla.com?content=example.dwollalabs.com`

Should the `?` be an `&`?

Reply:
You're definitely right. Don't no where my mind was on that one :)